### ISSUE TITLE
1.0.18 and static cpu data

### ISSUE THREAD

**Issue by ghost** [RevoGirl/RevoBoot#16]
Hi All

When I go for dynamic data collection for the CPU all compiles fine and works great.  When I try to compile for static cpu data I get the following error in terminal:
...
cc   -Os -Oz  -arch i386 -g -Wmost -D__ARCHITECTURE__=\"i386\" -DSAIO_INTERNAL_USER -DRCZ_COMPRESSED_FILE_SUPPORT -DNOTHING -DREVOBOOT_VERSION_INFO=\"RevoBoot\ v`cat ../../VERSION`\" -DMAKE_TARGET_OS=`echo 3;` -fno-builtin -static  -mpreferred-stack-boundary=2 -fno-align-functions -fno-stack-protector -march=pentium4 -msse2 -mfpmath=sse -msoft-float  -c -I. -I/Users/dave/RevoBoot/sym/i386 -I../util -I../libsa cpu.c -o /Users/dave/RevoBoot/obj/i386/libsaio/cpu.o \
        -MD -dependency-file /Users/dave/RevoBoot/obj/i386/libsaio/cpu.d
In file included from cpu.c:173:
cpu/intel/static_data.h: In function ‘initCPUStruct’:
cpu/intel/static_data.h:27: error: ‘struct CPU’ has no member named ‘MinBusRatio’
make[2]: **\* [cpu.o] Error 1
make[1]: **\* [all] Error 2
make: **\* [all] Error 2

Any ideas?

Thanks

Dave

**Comment by RevoGirl** [comment]
Yeah Jeroen forgot something. Let me solve this... Okay. Please pull again. Should be fine again.
